guide/ch_foreword.tex

```diff
@@ -1,199 +1,129 @@
+telescopes which became a popular feature. In the years to come this was occasionally 
+extended by further external contributors to provide support for INDI and RTS2 on Linux and ASCOM on Windows, 
+but keeping the systems working by occasionally upgrading the protocols was left to 
+the maintainers who may be limited in their access to instruments or just have 
+not even enough time to go observing themselves.  
+
+
+Only in late 2022 we were confident enough in the accuracy of our implementations of planetary 
+algorithms and various coordinate refinements that we have finally dropped the zero version number and 
+released version 1.0, and started using the year number as basis for later versions. 
+On this occasion we also moved forward to upgrading the foundational Qt GUI framework from Qt5 to Qt6 
+to be prepared for upgrades in the next years. To keep compatibility with older PCs, 
+we kept providing  builds based on the aging Qt5 framework. 
+However it turned out that the new Qt6 framework and the widely used telescope 
+control framework ASCOM~6 apparently don't play nicely together. We saw several reports about crashes coming in, 
+but were not able to really solve the issue, requiring us to finally disable support on Qt6-based builds. 
+The builds on the older Qt5 framework seemed to always work nicely. 
+
+%What we don't have currently is expertise in Goto telescopes. 
+
+
+With only three core developers working part-time at best and otherwise busy in other projects or daytime jobs, 
+we cannot do giant leaps, but the program has 
+evolved nicely in other areas like the AstroCalc dialog that becomes richer in every version. 
+
+
+
+While many improvements had involved  refinements for planetary positions and modelling of observing conditions, 
+other users were more interested in star positions over millennia. 
+The star catalog we have used since 2006, which was based on the 
+then best positional measurements made by ESA's Hipparcos satellite,  was discovered to have 
+a systematic error that caused slightly wrong coordinates millennia from the present. 
+Also, binary stars could be observed to fly apart, because their motions around each other were not modelled. 
+
+Just a few weeks before release 24.4 a new developer, Henry Leung, showed up and offered a new star catalog for Stellarium. 
+We needed a few weeks to test, check for inconsistencies and finding how to best link this to our existing infrastructure, 
+but days after 24.4 was out, we fully embraced this new catalog, now a combination of ESA's latest 
+and best astrometry satellite Gaia's DR3 catalog with -- still -- the Hipparcos data, 
+required for the brightest stars which were too bright for Gaia to measure. 
+Computers have become more powerful since 2006, and so this new catalog comes with new features: While previously 
+we just showed numerical data for parallax, now you can observe the stars' actual parallaxes as Earth moves around the Sun! 
```

Comment:
maybe change "Earth" to just "you"? Because you can observe parallax effect not just on Earth but on other planets/moons too